Src/WitsmlExplorer.Api/Services/CredentialsCache.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Caching;
+
+using Microsoft.Extensions.Logging;
+namespace WitsmlExplorer.Api.Services
+{
+
+    public interface ICredentialsCache
+    {
+        void Set(string cacheId, string encryptedCredentials, CacheItemPolicy policy);
```

Comment:
I am getting the following suggestion here
"Rename virtual/interface member ICredentialsCache.Set(string, string, CacheItemPolicy) so that it no longer conflicts with the reserved language keyword 'Set'. Using a reserved keyword as the name of a virtual/interface member makes it harder for consumers in other languages to override/implement the member. [WitsmlExplorer.Api]csharp(CA1716)"

Reply:
Probably, I did not get the same warning so thanks for the tip

Reply:
Changed to `Set/GetItem`